daemon/mgr/container_kill.go

```diff
@@ -0,0 +1,94 @@
+package mgr
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"syscall"
+
+	"github.com/pkg/errors"
+
+	"github.com/alibaba/pouch/pkg/errtypes"
+	"github.com/sirupsen/logrus"
+)
+
+// Kill kills a container.
+func (mgr *ContainerManager) Kill(ctx context.Context, name string, signal uint64) (err error) {
+	c, err := mgr.container(name)
+	if err != nil {
+		return err
+	}
+
+	if syscall.Signal(signal) == syscall.SIGKILL {
+		return mgr.kill(ctx, c)
+	}
+	return mgr.killWithSignal(ctx, c, int(signal))
+}
+
+func (mgr *ContainerManager) kill(ctx context.Context, c *Container) error {
+	if !c.IsRunning() {
+		return fmt.Errorf("Container %s is not running", c.ID)
+	}
+
+	if err := mgr.killDeadProcess(ctx, c, int(syscall.SIGKILL)); err != nil {
+		if errtypes.IsNoSuchProcess(err) {
+			return nil
+		}
+
+		if c.IsRunning() {
+			return err
+		}
+	}
+
+	if pid := c.State.Pid; pid != 0 {
```

Comment:
Please use `fast return` to reduce ident:
https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#rule006---return-fast-to-indent-less